pyproject.toml

```diff
@@ -47,7 +47,8 @@ croniter = ">=2.0"
 click = "^8.1"
 rq = "^1.16"
 pyyaml = { version = "^6.0", optional = true }
-valkey = "6.0.1"
+valkey = { version = "^6.0.2", optional = true}
+libvalkey = { version = "^4.0.0", optional = true}
```

Comment:
why is this needed?

Reply:
I think let's remove it, this package is not about the internals of how the connection is made and should obfuscate it. A user can obviously add it separately.